### PR TITLE
core/editor3: Delete & Resolve editor comments.

### PR DESCRIPTION
**Pull request by gbbr**

### CODE REVIEW

File: scripts/core/editor3/components/comments/CommentPopup.jsx [hunk 1]
```diff
@@ -58,12 +60,20 @@ class CommentPopupComponent extends Component {
...
+        const onReply = this.postReply.bind(this, comment.selection);
+        const onResolve = this.props.resolveComment.bind(this, comment);
+        const onDelete = () => ng.get('modal')
+            .confirm(gettext('The comment will be deleted. Are you sure?'))
+            .then(() => this.props.deleteHighlight(comment));
 
         return (
             <div>
                 <Comment data={comment.data} />
+                {isAuthor && <a className="btn btn--small btn--hollow" onClick={onDelete}>Delete</a>}
```
Comment (petrjasek) [inline scripts/core/editor3/components/comments/CommentPopup.jsx]: gettext missing on `Delete`

File: scripts/core/editor3/components/comments/CommentPopup.jsx [hunk 2]
```diff
@@ -58,12 +60,20 @@ class CommentPopupComponent extends Component {
...
+                {!resolved && <a className="btn btn--small btn--hollow" onClick={onResolve}>Resolve</a>}
```
Comment (petrjasek) [inline scripts/core/editor3/components/comments/CommentPopup.jsx]: gettext missing on `Resolve`

File: scripts/core/editor3/components/comments/CommentPopup.jsx [hunk 3]
```diff
@@ -73,7 +83,7 @@ class CommentPopupComponent extends Component {
                     <textarea ref={(el) => {
                         this.input = el;
                     }} />
-                    <a className="btn btn--small btn--hollow btn-reply" onClick={postReply}>Reply</a>
+                    <a className="btn btn--small btn--hollow btn-reply" onClick={onReply}>Reply</a>
```
Comment (petrjasek) [inline scripts/core/editor3/components/comments/CommentPopup.jsx]: and `Reply`

**Comment by gbbr** [comment]
@petrjasek PTAL

**Pull request merged**